packages/frontend/src/pages/project/components/sections/ContractsSection/ContractsSection.tsx

```diff
@@ -106,33 +111,42 @@ export function ContractsSection(props: ContractsSectionProps) {
           </div>
         </>
       )}
-      {props.nativeContracts.length > 0 && (
-        <>
-          <h3 className="font-bold">
-            The system consists of the following smart contracts on{' '}
-            {props.nativeChainName}:
-          </h3>
-          <div className="my-4">
-            {unchangedNativeContracts.map((contract, i) => (
-              <React.Fragment key={i}>
-                <ContractEntry
-                  contract={contract}
-                  verificationStatus={props.verificationStatus}
-                  manuallyVerifiedContracts={props.manuallyVerifiedContracts}
-                  className="my-4"
-                />
-              </React.Fragment>
-            ))}
-            {changedNativeContracts.length > 0 && (
-              <ImplementationHasChangedContracts
-                contracts={changedNativeContracts}
-                manuallyVerifiedContracts={props.manuallyVerifiedContracts}
-                verificationStatus={props.verificationStatus}
-              />
-            )}
-          </div>
-        </>
-      )}
+      {Object.keys(paritionedNativeContracts).length > 0 &&
+        Object.entries(paritionedNativeContracts).map(
+          ([chainName, [changedContracts, unchangedContracts]]) => {
+            return (
+              <div key={chainName}>
+                <h3 className="font-bold">
+                  The system consists of the following smart contracts on{' '}
+                  <span className="italic">{chainName}:</span>
```

Comment:
do we want to add italic here? host chain is not italic

Reply:
it kinda blends in, I don't know myself, added the italics just for it to pop, but if you noticed the discrepancy then maybe it's not worth it, i'll revert it